openstack/barbican/templates/seed.yaml

```diff
@@ -62,7 +62,7 @@ spec:
         role: cloud_keymanager_admin
     - name: barbican
       description: 'Barbican Service'
-      password: '{{.Values.global.barbican_service_password}}'
+      password: '{{.Values.global.barbican_service_password}} | include "resolve_secret"'
```

Comment:
Should be
'{{ .Values.global.barbican_service_password | include "resolve_secret"' }}'